src/main/resources/static/requestBlood.html

```diff
@@ -0,0 +1,315 @@
+
```

Comment:
shouldn't this page be restricted to verified users? If so, the login button and modal can be deleted

Reply:
that page is the old request blood, the new one is in the template folder